docs/postrelease.md

```diff
@@ -5,17 +5,14 @@ This checklist is to be run after cutting a release.
 ### All releases
 
 - [ ] Create PR to update default Teleport version in Teleport docs
-  - Example: https://github.com/gravitational/teleport/pull/4615
+  - Example: https://github.com/gravitational/teleport/pull/7033
 - [ ] Create PR to update default AMI versions in Makefile and AMIs.md under https://github.com/gravitational/teleport/blob/master/assets/aws
-  - Example: https://github.com/gravitational/teleport/pull/4608
-- [ ] Create PR to update app version and Helm chart version (https://github.com/gravitational/teleport/blob/master/examples/chart)
+  - Example command: `TELEPORT_VERSION=6.2.0 make -C assets/aws create-update-pr`
 
-### Major/minor releases only
+### Major releases only
 
 - [ ] Update support matrix in docs FAQ page
-- [ ] Create PR to update default Docker image version in Teleport docs
-  - Example: https://github.com/gravitational/teleport/pull/4615
```

Comment:
Is this no longer needed, or handled by a sibling step?

Reply:
It's now handled with a [variable include in the docs](https://github.com/gravitational/teleport/blame/branch/v6.2/docs/pages/quickstart-docker.mdx#L31-L34) from the [`docs/config.json` file](https://github.com/gravitational/teleport/blob/branch/v6.2/docs/config.json#L231-L240).